diff -urN ksudoku-0.2/src/ksudoku.desktop ksudoku-0.2.new/src/ksudoku.desktop
--- ksudoku-0.2/src/ksudoku.desktop	2005-08-28 12:31:27.000000000 +0200
+++ ksudoku-0.2.new/src/ksudoku.desktop	2005-09-21 15:44:34.000000000 +0200
@@ -1,9 +1,8 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=ksudoku
-Name[xx]=xxksudokuxx
 Exec=ksudoku %i %m -caption "%c"
-Icon=ksudoku
+Icon=ksudoku.png
 Type=Application
 DocPath=ksudoku/ksudoku.html
 Comment=A KDE KPart Application
@@ -39,3 +38,5 @@
 Comment[xx]=xxA KDE KPart Applicationxx
 Comment[zh_CN]=一个 KDE KPart 应用程序
 Terminal=false
+Categories=Qt;KDE;Game;BoardGame;
+# vi: encoding=utf-8
